Shared/Samples/Show device location with NMEA data sources/README.md

```diff
@@ -0,0 +1,76 @@
+# Show device location with NMEA data sources
+
+Parse NMEA sentences and use the results to show device location on the map.
+
+![Image of Show device location with NMEA data sources sample](show-device-location-with-NMEA-data-sources.png)
+
+## Use case
+
+NMEA sentences can be retrieved from an MFi GNSS/GPS accessory and parsed into a series of coordinates with additional information.
+
+The NMEA location data source allows for detailed interrogation of the information coming from a GNSS accessory. For example, allowing you to report the number of satellites in view, accuracy of the location, etc.
+
+## How to use the sample
+
+Tap "Source" to choose between a simulated location data source or any data source created from a connected GNSS device, and initiate the location display. Tap "Recenter" to recenter the location display. Tap "Reset" to reset the location display and location data source.
+
+## How it works
+
+1. Load NMEA sentences.
+    * If a supported GNSS accessory is connected, the sample can get NMEA updates from it.
+    * Otherwise, the sample will read mock data from a local file.
+2. Create an `NMEALocationDataSource`. There are 2 ways to provide updates to the data source.
+    * When updates are received from a GNSS accessory or the mock data provider, push the data into `NMEALocationDataSource`.
+    * You can initialize `NMEALocationDataSource` with a GNSS accessory. The data source created this way will automatically get updates from the accessory instead of requiring to push data explicitly.
+3. Set the `NMEALocationDataSource` to the location display's data source.
+4. Start the location display to begin receiving location and satellite updates.
+
+## Relevant API
+
+* Location
+* LocationDisplay
+* NMEALocationDataSource
+* NMEASatelliteInfo
+
+## About the data
+
+A list of NMEA sentences is used to initialize a `SimulatedNMEADataSource` object. This simulated data source provides NMEA data periodically and allows the sample to be used without a GNSS accessory.
```

Comment:
Was `SimulatedNMEADataSource` renamed to `SimulatedNMEASentenceFeed`?

Reply:
Great catch!